README.md

```diff
@@ -91,7 +90,7 @@ art = Artifactory(url="ARTIFACTORY_URL", auth=('USERNAME','PASSORD_OR_API_KEY'))
 
 Get the list of users:
 ```python
-users = art.users.list()
+users = art.users.list_all()
```

Comment:
Why `list_all()` and not just `list()` ?

Reply:
restored